server/validators/isSimpleTimeAfter.ts

```diff
@@ -0,0 +1,25 @@
+import { registerDecorator, ValidationOptions, ValidationArguments } from 'class-validator'
+import SimpleTime from '../routes/creatingLicences/types/time'
+
+export default function IsSimpleTimeAfter(property: string, validationOptions?: ValidationOptions) {
```

Comment:
Field decorator which takes in another SimpleTime object and compares it. Will pass if the field being validated is a time after the field its being compared against

```
earlierTime: SimpleTime

@IsSimpleTimeAfter('earlierTime', { message: 'nowTime should be after the earlierTime' })
nowTime: SimpleTime 
```